addons/auth_totp_mail_enforce/data/mail_template_data.xml

```diff
@@ -0,0 +1,52 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+    <data noupdate="1">
+        <record id="mail_template_totp_mail_code" model="mail.template">
+            <field name="name">TOTP for users: Authentication by email</field>
+            <field name="model_id" ref="base.model_res_users" />
+            <field name="subject">Your two-factor authentication code</field>
+            <field name="email_to">{{ object.email_formatted }}</field>
+            <field name="email_from">"{{ object.company_id.name }}" &lt;{{ (object.company_id.email or user.email) }}&gt;</field>
+            <field name="lang">{{ object.partner_id.lang }}</field>
+            <field name="auto_delete" eval="True"/>
+            <field name="body_html" type="html">
+<div style="margin: 0px; padding: 0px; font-size: 13px;">
+    <p style="margin: 0px; padding: 0px; font-size: 13px;">
```

Comment:
Probably should not be a paragraph, I don't think you can put a paragraph or a list inside a paragraph so I'd expect the rendering to be wonky.

Reply:
Well, this is actually a copy/paste from the existing `auth_totp_mail` module
https://github.com/odoo/odoo/blob/c93f7dba0720e5e4f7d0a26ce05a854094e6a279/addons/auth_totp_mail/data/mail_template_data.xml#L12-L14
Though I agree with you. So, removed.